internal/bsr/internal/fstest/fs.go

```diff
@@ -142,28 +142,44 @@ func (m *MemContainer) OpenFile(_ context.Context, n string, option ...storage.O
 		return nil, fmt.Errorf("cannot create file in read-only mode: %w", ErrReadOnly)
```

Comment:
Can you update the function doc for OpenFile to explain how opts.WithCreateFile works with the src and dst files?
(I think the supported ops is also out of date)